core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -22,6 +22,7 @@ import joptsimple._
 import joptsimple.util.EnumConverter
 import kafka.security.authorizer.AclAuthorizer
 import kafka.utils._
+import org.apache.kafka.common.utils.Exit
```

Comment:
@frankvicky How did you reorder the imports? The diff looks a lot larger now.

IMHO we can just insert the changed imports "close enough" to the correct order, and leave most of this as-is. Separately we could try and apply spotless in scala, (or leave it as-is until the scala gets removed)

@mimaison what did you have in mind here?

Reply:
Hi @gharris1727 
I have leveraged the function from IDEA while preserving the original imports if they were specific. If it was a wildcard import, I left it as it was. 👀

Reply:
I don't think we should reorder all imports, that just creates noise.
But here let's add the import with the other `import org.apache.kafka.common.utils` imports we have below instead of inserting at a pretty random place.

Reply:
I see.
I will do it in next commit 😃

Reply:
There are many import chaos in scala code, and that is why I do thank @gongxuanzhang to complete the old but important [KAFKA-10787](https://issues.apache.org/jira/browse/KAFKA-10787) which reducing the import conflicts of java code in the future.